libraries/chain/message_handling_contexts.cpp

```diff
@@ -18,7 +18,7 @@ void message_validate_context::require_authorization(const types::AccountName& a
 
 void message_validate_context::require_scope(const types::AccountName& account)const {
    auto itr = boost::find_if(trx.scope, [&account](const auto& scope) {
-      return scope == account;
```

Comment:
as you defined, the require_scope function's return type is "void", but "return scope == account;" means return a bool value, so if some wrong happen, should raise a exception, otherwise should just "return;" instead of "return true/false", pls confirm again.thx.